### PR TITLE
Add Tenro to LLM & MCP Testing

### PR DESCRIPTION
**Pull request by tenro-oss**
Adding [Tenro](https://github.com/tenro-ai/tenro-python) to the LLM & MCP Testing section.

Tenro is an open-source testing framework for AI agents. It simulates LLM responses and tool calls, making it possible to test edge cases, failure paths, and agent logic without live API calls. It also provides execution tracing to help with debugging.

- Provider-agnostic (OpenAI, Anthropic, Gemini)
- Integrates with pytest
- Apache 2.0 license

## Summary by Sourcery

Documentation:
- Document Tenro as an open-source testing framework for AI agents that simulates LLM and tool calls for robust testing without live API usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ Collection of awesome Python resources for testing and generating test data.
 ## LLM & MCP Testing
 
 - [mcp-server-fuzzer](https://github.com/Agent-Hellboy/mcp-server-fuzzer) - A comprehensive fuzzing tool designed specifically for testing Model Context Protocol (MCP) servers. It supports both tool argument fuzzing and protocol type fuzzing across multiple transport protocols.
+- [Tenro](https://github.com/tenro-ai/tenro-python) - Open-source testing framework for AI agents. Simulate LLM and tool calls to test edge cases, failure paths, and agent logic without live API calls.
```
[the remark this answers is not in the document]
Comment (tenro-oss) [reply README.md]: Updated description. Thanks!
